### DISCUSSION TITLE
firewall{,-offline}-cmd --version can only be executed by root

### DISCUSSION THREAD

**Discussion by unknowndevQwQ** [firewalld/firewalld#1412] [Q&A] [answered]
I don't think this is a reasonable restriction -- the `--help` option doesn't require root. If there is any special safety reason, please explain in detail.

**Comment by erig0** [comment, marked as answer]
There is no reason. File a bug.

**Reply by unknowndevQwQ** [reply]
Hey, could you please convert this into an issue?

**Reply by erig0** [reply]
Done as #1562 .

**Comment by erig0** [comment]
Closing as this has been converted into an Issue (#1562).
